src/services/asset-inventory/service-account/modules/ServiceAccountCredentialsForm.vue

```diff
@@ -155,10 +155,20 @@ export default defineComponent<Props>({
             providerData: {} as ProviderModel,
             showTrustedAccount: computed<boolean>(() => state.providerData?.capability?.support_trusted_service_account ?? false),
             trustedAccounts: [] as ServiceAccountModel[],
-            trustedAccountMenuItems: computed<SelectDropdownMenu[]>(() => state.trustedAccounts.map(d => ({
-                name: d.service_account_id,
-                label: d.name,
-            }))),
+            trustedAccountMenuItems: computed<SelectDropdownMenu[]>(() => {
+                const results: SelectDropdownMenu[] = [];
+                const baseInfoProperties: Record<string, JsonSchema> = state.baseInformationSchema?.properties;
+                state.trustedAccounts.forEach((d) => {
+                    let label = d.name;
+                    if (baseInfoProperties) {
+                        Object.entries(baseInfoProperties).forEach(([k, v]) => {
+                            label += ` | ${v.title}: ${d.data[k]}`;
+                        });
+                    }
+                    results.push({ name: d.service_account_id, label });
+                });
+                return results;
+            }),
```

Comment:
```suggestion
                const baseInfoProperties: Record<string, JsonSchema> = state.baseInformationSchema?.properties;
                const entries = Object.entries(baseInfoProperties)
                state.trustedAccounts.forEach((d) => {
                    let label = d.name;
                    if (baseInfoProperties) {
                        entries.forEach(([k, v]) => {
                            label += ` | ${v.title}: ${d.data[k]}`;
                        });
                    }
                    results.push({ name: d.service_account_id, label });
                });
                return results;
            }),
```

별건 아니지만 forEach 문 안에서 entries 돌리면 n 제곱이라서... 바깥으로 뺐으면 합니다~

Reply:
어이쿠 넵!!!!!